src/app/errors.rs

```diff
@@ -107,6 +108,9 @@ fn map_err_to_5xx(e: Error) -> StatusCode {
     } else if let Some(_) = e.downcast_ref::<shed::Overloaded>() {
         warn!("server overloaded, max-in-flight reached");
         http::StatusCode::SERVICE_UNAVAILABLE
+    } else if let Some(_) = e.downcast_ref::<buffer::Aborted>() {
+        warn!("dispatch aborted");
```

Comment:
Should this message say a little more? Such a terse message could be confusing in logs.